packages/vm/test/retesteth/README.md

```diff
@@ -10,7 +10,7 @@ Note: All paths are relative paths to the `VM` package root.
 
 1. If you change the port number in `transition-cluster.ts` to anything other than 3000 (or run `transition-cluster` on a separate machine or different IP address from retesteth), update `test/vm/retesteth/clients/ethereumjs/start.sh` to reflect the right IP and port.
 
-2. From VM package root directory, run `npx ts-node test/retesteth/transition-cluster.ts`
+2. From VM package root directory, run `npx ts-node test/retesteth/transition-cluster.cts`
```

Comment:
This is not relevant for this PR, but added it here, since it felt silly to open another PR for this.